scripts/ingests/ingest_wise_1810-1010.py

```diff
@@ -0,0 +1,115 @@
+    dec_1010=Angle("-10 10 00.5", u.degree).degree
+
+    ingest_sources(db, ["CWISEP J181006.00-101001.1"], references=["Schn20"], 
+               ras= [ra_1810], 
+               decs=[dec_1010], 
+               search_db=False)
+
+
+    #  Ingest other name for Wise 1810-1010 (one used in SIMBAD)
+    #  code from deprecated utils does not work
+    ingest_names(db, 'CWISEP J181006.00-101001.1', 'Wise 1810-1010')
+
+def add_parallaxes(db):
+    ingest_parallaxes(db, 
+                  sources = ["CWISEP J181006.00-101001.1"], 
+                  plxs = [112.5], 
+                  plx_errs = [8.1], 
+                  plx_refs = "Lodi22", 
+                  comments=None)
+
+def add_proper_motions(db):
+    ingest_proper_motions(db, sources = ["CWISEP J181006.00-101001.1"], 
+                          pm_ras = [-1027], 
+                          pm_ra_errs = [3.5],
+                          pm_decs = [-246.4], 
+                          pm_dec_errs = [3.6], 
+                          pm_references = "Schn20")
+
+
+#Ingest Functions for Modeled Parameters (Gravity, Metallicity, Radius, Mass)
+#Creating list of dictionaries for each value formatted for modeled parameters
+def add_modeled_parameters_dict(db):
+    ingest_modeled_parameters = [{ 
+                                    'Gravity':
+                                       {'value': 5.0,
+                                        'value_error': 0.25,
+                                        'parameter': "log g", 
+                                        'unit': 'dex',
+                                        'reference': "Lodi22"}, 
+
+                                   'Metallicity':
+                                       {'value': -1.5,
+                                        'value_error': 0.5,
+                                        'parameter': "metallicity", 
+                                        'unit': 'dex',
+                                        'reference': "Lodi22"},
+
+                                   'Radius':
+                                       {'value': 0.067,
+                                        'value_error': 0.032, #Highest value error was picked between +0.032 & -0.020 listed
+                                        'parameter': "radius", 
+                                        'unit': 'R_jup',
+                                        'reference': "Lodi22"},
+
+                                   'Mass':
+                                       {'value': 17,
+                                        'value_error': 56, #Highest value error was picked between +56 & -12 listed
+                                        'parameter': "mass", 
+                                        'unit': 'M_jup',
+                                        'reference': "Lodi22"},
+
+                                    'Effective Temperature':
+                                       {'value': 1000,
+                                        'parameter': "temperature", 
```

Comment:
you have "temperature".  Look at the Parameters table to see what the parameter name should be.

Reply:
also add the value_error.